CMakeLists.txt

```diff
@@ -377,40 +377,53 @@ if(WITH_VTK)
   endif()
 endif()
 
-#VTK can depend on Qt and search for its required version, so search after so we can overwrite Qt5_FOUND if the version we require is not found.
+#VTK can depend on Qt and search for its required version, so search after so we can overwrite Qt5_FOUND/Qt6_FOUND if the version we require is not found.
 option(WITH_QT "Build QT Front-End" TRUE)
 if(WITH_QT)
-  find_package(Qt5 5.9.5 COMPONENTS Concurrent OpenGL Widgets)
+  find_package(Qt6 QUIET COMPONENTS Concurrent OpenGL Widgets)
+  if (NOT Qt6_FOUND)
+    find_package(Qt5 5.9.5 QUIET COMPONENTS Concurrent OpenGL Widgets)
+    set(Qt_FOUND ${Qt5_FOUND})
+    if(Qt5_FOUND)
+      set(QT_VERSION_MAJOR 5)
```

Comment:
Thoughts on adding the following:
* `QT_VERSION = QT6_FOUND ? QT6_VERSION : QT5_VERSION`
* `QTX = "Qt${QT_VERSION_MAJOR}"`, QTX = Qt5 or Qt6 depending (rationale: Qt${QT_VERSION_MAJOR} seems like a high risk for being misspelled)